src/main/java/school/hei/haapi/model/validator/UserValidator.java

```diff
@@ -0,0 +1,36 @@
+package school.hei.haapi.model.validator;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.validation.ConstraintViolation;
+import javax.validation.Validator;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Component;
+import school.hei.haapi.model.User;
+import school.hei.haapi.model.exception.BadRequestException;
+
+@Component
+@AllArgsConstructor
+public class UserValidator {
```

Comment:
`implements Consumer<User>` so that the ugly name `validateSomeUser` becomes the standard `Consumer::accept`